packages/frontend/src/ui/remote-logo/index.tsx

```diff
@@ -26,12 +26,12 @@ export const RemoteLogo = ({
     src,
     address,
     chain,
-    size = "md",
+    size,
     defaultText = "?",
     className,
 }: RemoteLogoProps) => {
     const [imageError, setImageError] = useState(false);
-
+    const sizeClass = size ? styles[size] : styles.selfAdjust;
```

Comment:
Here I think we can avoid this and just apply both the classes and have the `selfAdjust` just apply the media query classes, something like:

```tsx
className={`${styles[size]} ${styles.selfAdjust} ${className}`}
```

```css
.selfAdjust {
    @apply md:w-[3.25rem] md:h-7;
}
```

Wdyt?

Reply:
But wouldn't this always prioritize the media queries? If we do this then we can only alter the sizes for small screens because that would be the only case that does not have any conflicts with media queries.

Reply:
Yeah no, you're right, let's keep it this way for now.